web/src/features/charts/ProductionSourceLegendList.tsx

```diff
@@ -16,12 +17,16 @@ export default function ProductionSourceLegendList({
   selectedData?: SelectedData;
   isDataInteractive?: boolean;
 }) {
+  const { t } = useTranslation();
   // TODO(cady): memoize
   return (
     <div className={twMerge('flex w-fit flex-row flex-wrap gap-1 py-1', className)}>
       {sources.map((source, index) => {
         const onClick = () => selectedData?.toggle(source);
-        const capitalizedLabel = `${source.charAt(0).toUpperCase()}${source.slice(1)}`;
+        const translatedSource = t(source.toLowerCase());
+        const capitalizedLabel = `${translatedSource
+          .charAt(0)
+          .toUpperCase()}${translatedSource.slice(1)}`;
```

Comment:
Not sure if this is relevant but should we really capitalize these?

Wouldn't the grammar and convention for this differ by language?

(this might be a total non issue in reality though so might be better to keep it as is until someone complains)

Reply:
Yeah, I wondered that too - different languages do have different capitalization conventions. I'd need to spend a little bit more thinking about a good approach. In the meantime, I think it's probably worth pushing this out so users can read the labels in their preferred language(s)